src/parse/parse_step.go

```diff
@@ -213,9 +213,11 @@ func parsePackage(state *core.BuildState, label, dependent core.BuildLabel, subr
 		}
 	}
 
-	// Verify some details of the output files in the background. Don't need to wait for this
-	// since it only issues warnings sometimes.
-	go pkg.VerifyOutputs()
+	if !state.Config.FeatureFlags.PackageOutputsStrictness {
+		// Verify some details of the output files in the background. Don't need to wait for this
+		// since it only issues warnings sometimes.
+		go pkg.VerifyOutputs()
+	}
```

Comment:
The only reason this was done here instead of on registering outputs was for performance wasn't it? Can we can just make this change now considering it's not a breaking change?

Reply:
I think the reason was also that `pkg.VerifyOutputs` would only output warnings, so it wasn't critical to put the logic where the outputs get registered.

If we make this change now, builds will error out in cases where warnings were dismissed and not handled. Not sure if this is considered a breaking change or patch one.